extensions/markdown/syntaxes/markdown.tmLanguage

```diff
@@ -3318,7 +3318,7 @@
 				<dict>
 					<key>begin</key>
 					<string>(?x)
-								(\*|_)(?=\S)								# Open
```

Comment:
Can you please update `markdown.tmLanguage.base` as well. We use that file to generate this one. You can run the generator using `(cd extensions/markdown; npm run update-grammar)`

Reply:
line 3371 must be changed from 
`<string>(?&lt;=\S)(\1)((?!\1)|(?=\1\1))</string>`
to
`<string>(?&lt;=\S)(\1)((?!\1)|(?=\1\1))\b</string>`

Reply:
Thank you for review. I will fix that today